Loop/Managers/LoopDataManager.swift

```diff
@@ -488,6 +488,26 @@ extension LoopDataManager {
             }
         }
     }
+    
+    /// Take actions to address how insulin is delivered when the CGM data is unreliable
+    ///
+    /// An active high temp basal (greater than the basal schedule) is cancelled when the CGM data is unreliable.
+    func recievedUnreliableCGMReading() {
+        guard case .tempBasal(let tempBasal) = basalDeliveryState,
+              let scheduledBasalRate = basalRateSchedule?.value(at: now()),
+              tempBasal.unitsPerHour > scheduledBasalRate else
+        {
+            return
+        }
+              
+        //TODO cancel temp basal.... is this possible here?
+        let basalRecommendation = TempBasalRecommendation(unitsPerHour: scheduledBasalRate, duration: .minutes(30))
```

Comment:
Seems like there should be a way to cancel any temp basal and resort back to scheduled basal.

Reply:
Yes, the way to cancel a temp basal is to pass a duration of 0.  We do not want to take the current scheduled basal rate and issue a 30m temp basal for it.